piksi_tools/console/baseline_view.py

```diff
@@ -208,23 +215,36 @@ def baseline_callback(self, sbp_msg):
     self.ns.append(soln.n)
     self.es.append(soln.e)
     self.ds.append(soln.d)
-
-    self.ns = self.ns[-1000:]
-    self.es = self.es[-1000:]
-    self.ds = self.ds[-1000:]
-
-    self.plot_data.set_data('n', self.ns)
-    self.plot_data.set_data('e', self.es)
-    self.plot_data.set_data('d', self.ds)
-    self.plot_data.set_data('curr_n', [soln.n])
-    self.plot_data.set_data('curr_e', [soln.e])
-    self.plot_data.set_data('curr_d', [soln.d])
+    self.fixeds.append(fixed)
+    
+    # Delete oldest entries to maintain no more than N in plot
+    self.ns = self.ns[-self.plot_history_max:]
+    self.es = self.es[-self.plot_history_max:]
+    self.ds = self.ds[-self.plot_history_max:]
+    self.fixeds = self.fixeds[-self.plot_history_max:]
+
+    self.plot_data.set_data('n_fixed', np.array(self.ns)[np.where(self.fixeds)])
+    self.plot_data.set_data('e_fixed', np.array(self.es)[np.where(self.fixeds)])
+    self.plot_data.set_data('d_fixed', np.array(self.ds)[np.where(self.fixeds)])
+    floats = np.logical_not(self.fixeds)
+    self.plot_data.set_data('n_float', np.array(self.ns)[floats])
+    self.plot_data.set_data('e_float', np.array(self.es)[floats])
+    self.plot_data.set_data('d_float', np.array(self.ds)[floats])
```

Comment:
I didn't, but I was testing on a PC with plenty of CPU.  I'll run it with htop on the Atom tablet and see how it looks.